src/chrome/content/rules/graylady3jvrrxbe.onion.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	For more NY Times ruleset, see NYTimes.xml
+
+-->
+<ruleset name="graylady3jvrrxbe.onion">
+
+	<target host="a1.graylady3jvrrxbe.onion" />
+		<test url="http://a1.graylady3jvrrxbe.onion/fonts/family/cheltenham/cheltenham-normal-400.ttf" />
+	<target host="typeface.graylady3jvrrxbe.onion" />
+		<test url="http://typeface.graylady3jvrrxbe.onion/fonts/nyt-cheltenham-extra-condensed-bold-700-normal.woff" />
```

Comment:
Please add


```
<target host="static01.graylady3jvrrxbe.onion" />
<test url="http://static01.graylady3jvrrxbe.onion/images/2017/10/28/business/28ECON1/27ECON1-largeHorizontalJumbo.jpg" />
```

Reply:
Thanks! Added.

Reply:
`^` and `www` redirect to `http`.

Reply:
@Bisaloo Didn't notice the redirect, thanks for the remark!